client/src/components/ManageArtist/ManagePost.tsx

```diff
@@ -75,7 +75,7 @@ const ManagePost: React.FC<{}> = () => {
                 align-items: center;
               `}
             >
-              <Link to={`/post/${post.id}`}>
+              <Link to={`/${artist?.urlSlug}/posts/${post.id}`}>
```

Comment:
There's a helper function called `getPostURLReference` that would make it so that this is consistent, but it might need a little rewrite to work with a potentially missing artist. You can see it in ArtistPosts.tsx

Reply:
Oh yeah, thanks for pointing that out! Just took a peek at the code and it looks like `getPostURLReference` is taking into account a potentially missing artist. Since this is merged, I can make a new PR to swap to that.